<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileModifier</class>
 <widget class="QWidget" name="FileModifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileModifier</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutSettings" rowstretch="0,0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="4">
     <widget class="QLabel" name="lbDeleteInputFile">
      <property name="text">
       <string>Delete input File</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="lbTimer">
      <property name="text">
       <string>Timer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="lbInpuFilePath">
      <property name="text">
       <string>Input file path</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="chckBDeleteInputFile">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="lbRepeatingFile">
      <property name="text">
       <string>Repeating Files</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="lbMaskOfInput">
      <property name="text">
       <string>Mask of input</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="lEinputPath">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="lbMaskOfOutput">
      <property name="text">
       <string>Mask of output</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pbtnBrowseInputPath">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons.qrc">
        <normaloff>:/icons/Icons/folderBrowserIcon.png</normaloff>:/icons/Icons/folderBrowserIcon.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <layout class="QHBoxLayout" name="layoutRepearingFiles">
      <item>
       <widget class="QRadioButton" name="rBtnModify">
        <property name="text">
         <string>Modify</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtnRewrite">
        <property name="text">
         <string>Rewrite</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="5">
     <widget class="QSpinBox" name="sBTimerDelay">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="lbTimerQuantity">
      <property name="text">
       <string>sec</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="lEMaskOutput">
      <property name="text">
       <string>*.txt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="lEMaskInput">
      <property name="text">
       <string>*.dat</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbtnRun">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbtnStop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
